### DISCUSSION TITLE
Audio Gridder Cross platform?

### DISCUSSION THREAD

**Discussion by BubsonGH** [apohl79/audiogridder#651] [Q&A] [answered]
Hi I am just wondering if I can put audio gridder server on a mac while running the plugin on a windows system? thanks

**Comment by apohl79** [comment, marked as answer]
you can

**Reply by BubsonGH** [reply]
Thanks, Could you point me to the full setup manual? I cant seem to find it

**Comment by CaBuffalo** [comment]
Yes, you can.

On Tue, Jul 20, 2021 at 7:32 AM Andreas Pohl ***@***.***>
wrote:

> you can
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/651#discussioncomment-1026689>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DQPEJBMGTFD5AOGATTYWCJDANCNFSM5AVIMYFQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by BubsonGH** [reply]
Thank you

**Comment by CaBuffalo** [comment]
There is no manual. The installation is very easy and straightforward. You
can find more information here or you can email us.

https://audiogridder.com/plugin/
https://audiogridder.com/server/

On Tue, Jul 20, 2021 at 8:28 AM BubsonGH ***@***.***> wrote:

> Thank you
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/651#discussioncomment-1026970>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AXDAP24BKB3OHH4CTTYWIYTANCNFSM5AVIMYFQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by BubsonGH** [reply]
Thanks, tbh I just wanted to take a look at the options of how to connect the 2 computers together if I decided to grab a mac. But to be honest I primarily wanted to test if the software would work locally on the same computer as I am running a 5950x, after some testing it seems to work flawlessly!

**Reply by BubsonGH** [reply]
just posted about it here https://github.com/apohl79/audiogridder/discussions/654
